chia/protocols/protocol_state_machine.py

```diff
@@ -58,3 +58,8 @@ def sent_message_response_ok(sent: ProtocolMessageTypes, received: Optional[Prot
             return False
 
     return True
+
+
+def main():
+    # Check this static invariant at import time
+    static_check_sent_message_response()
```

Comment:
The need for this will go away when we remove `VALID_REPLY_MESSAGE_MAP`, and use only the `reply_type` decerator.

Reply:
does anyone call `main()`? The comment seem to suggest that it's called at import time, but I don't see it.
Did you mean to put this call in the global scope?